### DISCUSSION TITLE
Dark mode on nextjs docos

### DISCUSSION THREAD

**Discussion by udaykor** [vercel/next.js#36161] [Ideas]
## Dark mode

Dear Vercel+NextJS Maintainers, 

I would like to request for dark-mode on nextjs docs website. Please help consider the request.

Site: https://nextjs.org/

## Similarity comparison

Vercel has three modes - System | Dark | Light. It would be awesome to have the same here as well. 

## Alternatives

1. Work with lowest brightness possible on my screens.

**Comment by abdurrahmanseyidoglu** [comment]
I'm using an extension called "Dark Mode" on chrome, there is similar extensions for Firefox and opera as well

**Comment by florian-lefebvre** [comment]
I'm also interested in this
